### DISCUSSION TITLE
Acelle mail - [InvalidServerAPIKey] The API token provided in X-Server-API-Key was not valid.

### DISCUSSION THREAD

**Discussion by jpcapone** [postalserver/postal#2453] [Help with using Postal]
I get this error `[InvalidServerAPIKey] The API token provided in X-Server-API-Key was not valid.` when sending a test message via Acelle mail.  I can confirm that Acelle mail connects to the server running Postal as the test connection button returns a positive result.  Any suggestions would be appreciated.

**Comment by willpower232** [comment]
Sounds like you may be using an SMTP key for the API?
